$MirOS$

	Octets and Multibyte Characters are handled the same
	in zsh on Debian, so we must do so in MirBSD as well

--- Src/builtin.c.orig	Wed May 20 09:47:58 2009
+++ Src/builtin.c	Sun Sep  6 16:00:32 2009
@@ -4239,6 +4239,10 @@ bin_print(char *name, char **args, Optio
 			mb_metacharinit();
 			(void)mb_metacharlenconv(metafy(curarg+1, curlen-1,
 							META_USEHEAP), &cc);
+#ifdef iswoctet
+			if (iswoctet(cc))
+				cc &= 0xFF;
+#endif
 		    }
 		    else
 			cc = WEOF;
